src/pages/changelog/changelog.js

```diff
@@ -1,17 +1,19 @@
-const { app } = require('electron').remote
+const { ipcRenderer } = require('electron')
 const markdown = require('markdown').markdown
 const fetch = require('node-fetch')
 const settingsProvider = require('../../providers/settingsProvider')
 
 fetch(`https://api.github.com/repos/ytmdesktop/ytmdesktop/releases`)
     .then((res) => res.json())
     .then((json) => {
-        let tag = `v${app.getVersion()}`
+        let tag = ipcRenderer.sendSync('get-app-version')
```

Comment:
update on this, The Versions of 1.14.X which I have released did not include the `v` in them however otherwise they have previously.
So yes, Please make sure that either the v is prepended either in `get-app-version` OR in a string handle here.